### PR TITLE
add config

### PR DESCRIPTION
**Pull request by wuyefeilin**
add config
1. add num_classes, backbone, model_pretrained for models
2. add init_weight function for all model and backbone
3. add backbone_pretrained for backbone
4. return list of backbone
5. create basic config file
6. separate loss and set it from config file
7. init params in init_weight function.

### CODE REVIEW

File: dygraph/configs/fcn_hrnet/fcn_hrnetw18_cityscapes_1024x512_100k.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+_base_: '../_base_/cityscapes.yml'
```
Comment (nepeplwu) [inline dygraph/configs/fcn_hrnet/fcn_hrnetw18_cityscapes_1024x512_100k.yml:1]: base建议是一个列表

File: dygraph/cvlibs/param_init.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import paddle.fluid as fluid
+
+
+def constant_init(param, value=0.0):
```
Comment (nepeplwu) [inline dygraph/cvlibs/param_init.py]: 建议这么写
```python
def constant_init(param, **kwargs):
    initializer = fluid.initializer.Constant(**kwargs)
    initializer(param, param.block)
```

File: dygraph/models/fcn.py [hunk 3]
```diff
@@ -33,114 +34,133 @@
...
-            axis=-1)
-
-        loss = loss * mask
-        avg_loss = fluid.layers.mean(loss) / (
-            fluid.layers.mean(mask) + self.EPS)
-
-        label.stop_gradient = True
-        mask.stop_gradient = True
-        return avg_loss
+    # def _get_loss(self, logit, label):
```
Comment (nepeplwu) [inline dygraph/models/fcn.py]: 直接去掉

File: dygraph/cvlibs/param_init.py (hunk 2 above)
Comment (wuyefeilin) [reply dygraph/cvlibs/param_init.py]: 已更改
